schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_polymer_entity.json

```diff
@@ -4728,15 +5161,29 @@
                "context": "dictionary"
             }
          ]
+      },
+      "rcsb_membrane_lineage_provenance_code": {
+         "type": "string",
+         "enum": [
+            "Homology",
+            "Mpstruc"
+         ],
+         "description": "Mpstruc keyword denotes original annotation, Homology keyword denotes annotation inferred by homology.",
+         "rcsb_description": [
+            {
+               "text": "Mpstruc keyword denotes original annotation, Homology keyword denotes annotation inferred by homology.",
+               "context": "dictionary"
+            }
+         ]
```

Comment:
"rcsb_polymer_entity_container_identifiers" in `required` is missing

Reply:
@valasatava Do we also want to make this required for nonpolymer and branched entities?